### PR TITLE
LYNC Collection Claimer Template

### PR DESCRIPTION
**Pull request by shanu12joshi**
LYNC Collection Claimer Template allows developers to build their own Claimer page via LYNC Template by passing a few parameters.

### CODE REVIEW

**Comment by rekpero** [comment]
@shomix please check this PR and merge it if you think everything is fine

File: package.json [hunk 1]
```diff
@@ -11,9 +11,8 @@
     "lint": "turbo run lint"
   },
   "devDependencies": {
-    "turbo": "^1.7.3"
+    "turbo": "^1.9.3"
```
Comment (shomix) [inline package.json:14]: @shanu12joshi is there any reason for increasing the version of turbo?

File: package-lock.json
Comment (shomix) [inline package-lock.json:1]: Are the changes in package-lock.json needed?

**Review by shomix — COMMENTED** [review]
Hmm, from what I see only the changes in `packages/cli/src/commands/create-dapp.ts` are needed?

File: package.json (hunk 1 above)
Comment (shanu12joshi) [reply package.json:14]: Hey, @shomix Just tested with "turbo": "^1.7.3" and it works. No need to merge this file.

File: package-lock.json
Comment (shanu12joshi) [reply package-lock.json:1]: Not needed

File: package.json (hunk 1 above)
Comment (shomix) [reply package.json:14]: Cool!
Can you revert this files, after which I'll merge and publish this :)

File: package-lock.json
Comment (rekpero) [reply package-lock.json:1]: Please remove this file

Comment (shanu12joshi) [reply package-lock.json:1]: okay

**Pull request merged**